contrib/opentimelineio_contrib/adapters/ale.py

```diff
@@ -126,6 +154,28 @@ def _parse_data_line(line, columns, fps):
             line, repr(ex)
         ))
 
+def _cdl_values_from_metadata(asc_sop_string):
+
+    if not isinstance(asc_sop_string, (str, unicode)):
```

Comment:
`unicode` and `str` are just merged to `str` in python3 so the `isinstance` check is problematic there. In what cases will this be a non-string type? Perhaps we can get away with something like an `is None` check here?

Reply:
Oh I see!
This is basically just me trying to do type checking in python, I don't know in which cases this value will not be a string, but I have no guarantee that it will always be - that's why I put that check in place as a "defence" mechanism - the idea being that the sooner and you raise an Exception, the better (and also explicit is always better than implicit).
I guess it could become a check for None, but it would also become a little bit less explicit.
The alternative would be adding a check like this  https://stackoverflow.com/a/32632271 ?
If we feel safe enough about `metadata['ASC_SOP']` and `metadata['CDL']` always being strings or `None`, we could just actually check for `None`. What do you think?

PS: linting fixed!

Reply:
You can use `isinstance(foo, (type(""), type(u"")))` to get around the issue.

Reply:
@jminor Nice trick!